fvm/blueprints/system.go

```diff
@@ -51,3 +48,36 @@ func SystemChunkTransaction(chain flow.Chain) (*flow.TransactionBody, error) {
 
 	return systemTxBody, nil
 }
+
+// SystemCollection returns the re-created system collection after it has been already executed
```

Comment:
I think we can just remove this entirely from blueprints and use the version from access model instead.